packages/lwc-language-server/src/lwc-server.ts

```diff
@@ -116,6 +116,7 @@ export default class Server {
                 textDocumentSync: this.documents.syncKind,
                 completionProvider: {
                     resolveProvider: true,
+                    triggerCharacters: ['-', '_', '<', '"', '=', '/', '>'],
```

Comment:
should `'.'` be included too? https://microsoft.github.io/language-server-protocol/specification#textDocument_completion

Reply:
That's a good question, I initially had it but I couldn't find an example to justify having that character around. In which situations would we have aura or lwc refs after a `.` character ?

Reply:
I mentioned it since the above doc uses `.` as an example.
> If code complete should automatically be trigger on characters not being valid inside an identifier (for example `.` in JavaScript) list them in `triggerCharacters`.

> In which situations would we have aura or lwc refs after a . character ?

For example data binding in html templates`{data.name}` https://lwc.dev/guide/html_templates#data-binding

Reply:
Yeah, that requires a different change which I'll take care of with W-8017495 but I've added those characters for now since it could also help in some js scenarios.